tests/unit/packaging/test_utils.py

```diff
@@ -27,11 +28,37 @@ def test_simple_detail_empty_string(db_request):
     FileFactory.create(release=release)
 
     db_request.route_url = lambda *a, **kw: "the-url"
+    db_request.find_service = lambda svc, name=None, context=None: {
+        IIntegrityService: pretend.stub(
+            get_provenance_digest=pretend.call_recorder(lambda f: None),
+        ),
+    }.get(svc)
+
     expected_content = _simple_detail(project, db_request)
 
     assert expected_content["files"][0]["requires-python"] is None
 
 
+def test_simple_detail_with_provenance(db_request):
+    project = ProjectFactory.create()
+    release = ReleaseFactory.create(project=project, version="1.0")
+    FileFactory.create(release=release)
+
+    hash_digest = "deadbeefdeadbeefdeadbeefdeadbeef"
+
+    db_request.route_url = lambda *a, **kw: "the-url"
+    db_request.find_service = pretend.call_recorder(
+        lambda svc, name=None, context=None: {
+            IIntegrityService: pretend.stub(
+                get_provenance_digest=pretend.call_recorder(lambda f: hash_digest),
+            ),
+        }.get(svc)
+    )
+
+    expected_content = _simple_detail(project, db_request)
+    assert expected_content["files"][0]["provenance"] == hash_digest
```

Comment:
Suggestion: I think it would make sense to use the integrity service here instead of mocking it. 

```python
def test_simple_detail_with_provenance(db_request, integrity_service):
    project = ProjectFactory.create()
    release = ReleaseFactory.create(project=project, version="1.0")
    file = FileFactory.create(release=release)
    file.attestations.append(AttestationFactory.create(file=file))

    db_request.route_url = lambda *a, **kw: "the-url"
    db_request.find_service = pretend.call_recorder(
        lambda svc, name=None, context=None: {
            IIntegrityService: integrity_service,
        }.get(svc)
    )

    expected_content = _simple_detail(project, db_request)
    assert isinstance(expected_content["files"][0]["provenance"], str)
```

Reply:
Makes sense to me, go ahead and make this change!